mkdocs.yml

```diff
@@ -19,7 +19,6 @@ pages:
         - 'Core':
             - 'Design': "dev/core/core.md"
         - 'REST API':
-            - 'Retrofit Wrapper': 'dev/core/retrofit-wrapper.md'
```

Comment:
Please readd this!

Reply:
Ou damn! I put myself into a bad place with checking branches out based on other branches.